<!DOCTYPE html>
<html lang="en-US">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Awesome Bioinformatics | Awesome Lists with GitHub stars</title>
<meta name="generator" content="Jekyll v3.9.3">
<meta property="og:title" content="Awesome Bioinformatics">
<meta property="og:locale" content="en_US">
<link rel="canonical" href="https://yz89122.github.io/awesome-with-stars/lists/danielecook-Awesome-Bioinformatics.html">
<meta property="og:url" content="https://yz89122.github.io/awesome-with-stars/lists/danielecook-Awesome-Bioinformatics.html">
<meta property="og:site_name" content="Awesome Lists with GitHub stars">
<meta property="og:type" content="website">
<meta name="twitter:card" content="summary">
<meta property="twitter:title" content="Awesome Bioinformatics">
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","headline":"Awesome Bioinformatics","url":"https://yz89122.github.io/awesome-with-stars/lists/danielecook-Awesome-Bioinformatics.html"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="stylesheet" href="/awesome-with-stars/assets/css/style.css?v=66e193ded4a5d1e3066646f1de9d08d74602c950">
    <!-- start custom head snippets, customize with your own _includes/head-custom.html file -->

<!-- Setup Google Analytics -->



<!-- You can set your favicon here -->
<!-- link rel="shortcut icon" type="image/x-icon" href="/awesome-with-stars/favicon.ico" -->

<!-- end custom head snippets -->

  </head>
  <body>
    <div class="container-lg px-3 my-5 markdown-body">
      
      <h1><a href="https://yz89122.github.io/awesome-with-stars/">Awesome Lists with GitHub stars</a></h1>
      

      <h1 id="awesome-bioinformatics---">Awesome Bioinformatics <a href="https://github.com/sindresorhus/awesome"><img src="https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg" alt="Awesome" referrerpolicy="no-referrer" loading="lazy"></a> <a href="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/url-check.yml"><img src="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/url-check.yml/badge.svg" alt="Check URLs ![GitHub Repo Stars](https://img.shields.io/github/stars/danielecook/Awesome-Bioinformatics) ![GitHub last commit](https://img.shields.io/github/last-commit/danielecook/Awesome-Bioinformatics)" referrerpolicy="no-referrer" loading="lazy"></a> <a href="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/toc.yml"><img src="https://github.com/danielecook/Awesome-Bioinformatics/actions/workflows/toc.yml/badge.svg" alt="Generate TOC ![GitHub Repo Stars](https://img.shields.io/github/stars/danielecook/Awesome-Bioinformatics) ![GitHub last commit](https://img.shields.io/github/last-commit/danielecook/Awesome-Bioinformatics)" referrerpolicy="no-referrer" loading="lazy"></a>
</h1>

<blockquote>
  <p>Bioinformatics is an interdisciplinary field that develops methods and software tools for understanding biological data. — <a href="https://en.wikipedia.org/wiki/Bioinformatics">Wikipedia</a></p>
</blockquote>

<p>A curated list of awesome Bioinformatics software, resources, and libraries. Mostly command line based, and free or open-source. Please feel free to <a href="CONTRIBUTING.md">contribute</a>!</p>

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
<h2 id="table-of-contents">Table of Contents</h2>

<ul>
  <li><a href="#package-suites">Package suites</a></li>
  <li>
<a href="#data-tools">Data Tools</a>
    <ul>
      <li><a href="#downloading">Downloading</a></li>
      <li><a href="#compressing">Compressing</a></li>
    </ul>
  </li>
  <li>
<a href="#data-processing">Data Processing</a>
    <ul>
      <li><a href="#command-line-utilities">Command Line Utilities</a></li>
    </ul>
  </li>
  <li>
<a href="#next-generation-sequencing">Next Generation Sequencing</a>
    <ul>
      <li><a href="#workflow-managers">Workflow Managers</a></li>
      <li><a href="#pipelines">Pipelines</a></li>
      <li><a href="#sequence-processing">Sequence Processing</a></li>
      <li><a href="#data-analysis">Data Analysis</a></li>
      <li>
<a href="#sequence-alignment">Sequence Alignment</a>
        <ul>
          <li><a href="#pairwise">Pairwise</a></li>
          <li><a href="#multiple-sequence-alignment">Multiple Sequence Alignment</a></li>
          <li><a href="#clustering">Clustering</a></li>
        </ul>
      </li>
      <li><a href="#quantification">Quantification</a></li>
      <li>
<a href="#variant-calling">Variant Calling</a>
        <ul>
          <li><a href="#structural-variant-callers">Structural variant callers</a></li>
        </ul>
      </li>
      <li><a href="#bam-file-utilities">BAM File Utilities</a></li>
      <li><a href="#vcf-file-utilities">VCF File Utilities</a></li>
      <li><a href="#gff-bed-file-utilities">GFF BED File Utilities</a></li>
      <li><a href="#variant-simulation">Variant Simulation</a></li>
      <li><a href="#variant-predictionannotation">Variant Prediction/Annotation</a></li>
      <li>
<a href="#python-modules">Python Modules</a>
        <ul>
          <li><a href="#data">Data</a></li>
          <li><a href="#tools">Tools</a></li>
        </ul>
      </li>
      <li><a href="#assembly">Assembly</a></li>
      <li><a href="#annotation">Annotation</a></li>
    </ul>
  </li>
  <li>
<a href="#long-read-sequencing">Long-read sequencing</a>
    <ul>
      <li><a href="#long-read-assembly">Long-read Assembly</a></li>
    </ul>
  </li>
  <li>
<a href="#visualization">Visualization</a>
    <ul>
      <li><a href="#genome-browsers--gene-diagrams">Genome Browsers / Gene Diagrams</a></li>
      <li><a href="#circos-related">Circos Related</a></li>
    </ul>
  </li>
  <li><a href="#database-access">Database Access</a></li>
  <li>
<a href="#resources">Resources</a>
    <ul>
      <li><a href="#becoming-a-bioinformatician">Becoming a Bioinformatician</a></li>
      <li><a href="#bioinformatics-on-github">Bioinformatics on GitHub</a></li>
      <li><a href="#sequencing">Sequencing</a></li>
      <li><a href="#rna-seq">RNA-Seq</a></li>
      <li><a href="#chip-seq">ChIP-Seq</a></li>
      <li><a href="#youtube-channels-and-playlists">YouTube Channels and Playlists</a></li>
      <li><a href="#blogs">Blogs</a></li>
      <li><a href="#miscellaneous">Miscellaneous</a></li>
    </ul>
  </li>
  <li><a href="#online-networking-groups">Online networking groups</a></li>
  <li><a href="#license">License</a></li>
</ul>

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

<hr>

<h2 id="package-suites">Package suites</h2>

<p>Package suites gather software packages and installation tools for specific languages or platforms. We have some for bioinformatics software.</p>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/bioperl/bioperl-live">Bioperl <img src="https://img.shields.io/github/stars/bioperl/bioperl-live" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/bioperl/bioperl-live" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - International association of users &amp; developers of open source Perl tools for bioinformatics, genomics and life sciences. [ <a href="https://doi.org/10.1101%2Fgr.361602">paper-2002</a>
</td>
          <td>
<a href="https://bioperl.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/Bioconductor">Bioconductor</a></strong> - A plethora of tools for analysis and comprehension of high-throughput genomic data, including 1500+ software packages. [ <a href="https://link.springer.com/article/10.1186/gb-2004-5-10-r80">paper-2004</a>
</td>
          <td>
<a href="https://www.bioconductor.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/biopython/biopython">Biopython <img src="https://img.shields.io/github/stars/biopython/biopython" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/biopython/biopython" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Freely available tools for biological computing in Python, with included cookbook, packaging and thorough documentation. Part of the <a href="http://open-bio.org/">Open Bioinformatics Foundation</a>. Contains the very useful <a href="https://biopython.org/DIST/docs/api/Bio.Entrez-module.html">Entrez</a> package for API access to the NCBI databases. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19304878">paper-2009</a>
</td>
          <td>
<a href="https://biopython.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/bioconda">Bioconda</a></strong> - A channel for the <a href="http://conda.pydata.org/docs/intro.html">conda package manager</a> specializing in bioinformatics software. Includes a repository with 3000+ ready-to-install (with <code class="language-plaintext highlighter-rouge">conda install</code>) bioinformatics packages. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29967506">paper-2018</a>
</td>
          <td>
<a href="https://bioconda.github.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/BioJulia">BioJulia</a></strong> - Bioinformatics and computational biology infastructure for the Julia programming language. [ <a href="https://biojulia.net">web</a> ]</li>
  <li>
<strong><a href="https://github.com/rust-bio/rust-bio">Rust-Bio <img src="https://img.shields.io/github/stars/rust-bio/rust-bio" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/rust-bio/rust-bio" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Rust implementations of algorithms and data structures useful for bioinformatics. [ <a href="http://bioinformatics.oxfordjournals.org/content/early/2015/10/06/bioinformatics.btv573.short?rss=1">paper-2016</a> ]</li>
  <li>
<strong><a href="https://github.com/seqan/seqan3">SeqAn <img src="https://img.shields.io/github/stars/seqan/seqan3" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/seqan/seqan3" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - The modern C++ library for sequence analysis.</li>
  <li>
<strong><a href="https://github.com/TimothyStiles/poly">(Poly)merase <img src="https://img.shields.io/github/stars/TimothyStiles/poly" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/TimothyStiles/poly" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A Go library and command line utility for engineering organisms.</li>
  <li>
<strong><a href="https://github.com/biocaml/biocaml">Biocaml <img src="https://img.shields.io/github/stars/biocaml/biocaml" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/biocaml/biocaml" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Biocaml aims to be a high-performance user-friendly library for Bioinformatics.</li>
</ul>

<h2 id="data-tools">Data Tools</h2>

<h3 id="downloading">Downloading</h3>
<ul>
  <li>
<strong><a href="https://github.com/gogetdata/ggd-cli">GGD <img src="https://img.shields.io/github/stars/gogetdata/ggd-cli" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/gogetdata/ggd-cli" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Go Get Data; A command line interface for obtaining genomic data. [ <a href="https://gogetdata.github.io">web</a> ]</li>
  <li>
<strong><a href="https://github.com/ewels/sra-explorer">SRA-Explorer <img src="https://img.shields.io/github/stars/ewels/sra-explorer" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ewels/sra-explorer" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Easily get SRA download links and other information. [ <a href="https://sra-explorer.info">web</a> ]</li>
</ul>

<h3 id="compressing">Compressing</h3>
<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/divonlan/genozip">Genozip <img src="https://img.shields.io/github/stars/divonlan/genozip" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/divonlan/genozip" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A compressor of common genomic file formats (BAM, CRAM, FASTQ, VCF etc). [ <a href="https://genozip.com/?utm_source=Awesome-Bioinformatics">web</a>
</td>
          <td>
<a href="https://www.researchgate.net/publication/349347156_Genozip_-_A_Universal_Extensible_Genomic_Data_Compressor">paper-2021</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h2 id="data-processing">Data Processing</h2>

<h3 id="command-line-utilities">Command Line Utilities</h3>

<ul>
  <li>
<strong><a href="https://github.com/stephenturner/oneliners">Bioinformatics One Liners <img src="https://img.shields.io/github/stars/stephenturner/oneliners" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/stephenturner/oneliners" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Git repo of useful single line commands.</li>
  <li>
<strong><a href="https://github.com/bionode/bionode">BioNode <img src="https://img.shields.io/github/stars/bionode/bionode" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/bionode/bionode" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Modular and universal bioinformatics, Bionode provides pipeable UNIX command line tools and JavaScript APIs for bioinformatics analysis workflows. [ <a href="http://bionode.io">web</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/bioSyntax/bioSyntax">bioSyntax <img src="https://img.shields.io/github/stars/bioSyntax/bioSyntax" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/bioSyntax/bioSyntax" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Syntax Highlighting for Computational Biology file formats (SAM, VCF, GTF, FASTA, PDB, etc…) in vim/less/gedit/sublime. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30134911">paper-2018</a>
</td>
          <td>
<a href="http://www.bioSyntax.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/wireservice/csvkit">CSVKit <img src="https://img.shields.io/github/stars/wireservice/csvkit" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/wireservice/csvkit" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Utilities for working with CSV/Tab-delimited files. [ <a href="https://csvkit.readthedocs.io/en/latest">web</a> ]</li>
  <li>
<strong><a href="https://github.com/shenwei356/csvtk">csvtk <img src="https://img.shields.io/github/stars/shenwei356/csvtk" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/shenwei356/csvtk" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Another cross-platform, efficient, practical and pretty CSV/TSV toolkit. [ <a href="https://bioinf.shenwei.me/csvtk">web</a> ]</li>
  <li>
<strong><a href="https://git.savannah.gnu.org/gitweb/?p=datamash.git">datamash</a></strong> - Data transformations and statistics. [ <a href="http://www.gnu.org/software/datamash">web</a> ]</li>
  <li>
<strong><a href="https://github.com/shenwei356/easy_qsub">easy_qsub <img src="https://img.shields.io/github/stars/shenwei356/easy_qsub" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/shenwei356/easy_qsub" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Easily submitting PBS jobs with script template. Multiple input files supported.</li>
  <li>
<strong>GNU Parallel</strong> - General parallelizer that runs jobs in parallel on a single multi-core machine. <a href="https://www.biostars.org/p/63816/">Here</a> are some example scripts using GNU Parallel. [ <a href="http://www.gnu.org/software/parallel">web</a> ]</li>
  <li>
<strong><a href="https://github.com/arq5x/grabix">grabix <img src="https://img.shields.io/github/stars/arq5x/grabix" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/arq5x/grabix" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A wee tool for random access into BGZF files.</li>
  <li>
<strong><a href="https://github.com/brentp/gsort">gsort <img src="https://img.shields.io/github/stars/brentp/gsort" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/gsort" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Sort genomic files according to a specified order.</li>
  <li>
<strong><a href="https://github.com/samtools/tabix">tabix <img src="https://img.shields.io/github/stars/samtools/tabix" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/samtools/tabix" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Table file index. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21208982">paper-2011</a> ]</li>
  <li>
<strong><a href="https://github.com/wormtable/wormtable">wormtable <img src="https://img.shields.io/github/stars/wormtable/wormtable" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/wormtable/wormtable" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Write-once-read-many table for large datasets.</li>
  <li>
<strong><a href="https://github.com/mattgodbolt/zindex">zindex <img src="https://img.shields.io/github/stars/mattgodbolt/zindex" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/mattgodbolt/zindex" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Create an index on a compressed text file.</li>
</ul>

<h2 id="next-generation-sequencing">Next Generation Sequencing</h2>

<h3 id="workflow-managers">Workflow Managers</h3>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/pcingola/BigDataScript">BigDataScript <img src="https://img.shields.io/github/stars/pcingola/BigDataScript" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pcingola/BigDataScript" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A cross-system scripting language for working with big data pipelines in computer systems of different sizes and capabilities. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25189778">paper-2014</a>
</td>
          <td>
<a href="https://pcingola.github.io/BigDataScript">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/ssadedin/bpipe">Bpipe <img src="https://img.shields.io/github/stars/ssadedin/bpipe" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ssadedin/bpipe" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A small language for defining pipeline stages and linking them together to make pipelines. [ <a href="http://docs.bpipe.org">web</a> ]</li>
  <li>
<strong><a href="https://github.com/common-workflow-language/common-workflow-language">Common Workflow Language <img src="https://img.shields.io/github/stars/common-workflow-language/common-workflow-language" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/common-workflow-language/common-workflow-language" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - a specification for describing analysis workflows and tools that are portable and scalable across a variety of software and hardware environments, from workstations to cluster, cloud, and high performance computing (HPC) environments. [ <a href="http://www.commonwl.org">web</a> ]</li>
  <li>
<strong><a href="https://github.com/broadinstitute/cromwell">Cromwell <img src="https://img.shields.io/github/stars/broadinstitute/cromwell" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/broadinstitute/cromwell" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A Workflow Management System geared towards scientific workflows. [ <a href="https://cromwell.readthedocs.io">web</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/galaxyproject">Galaxy</a></strong> - a popular open-source, web-based platform for data intensive biomedical research. Has several features, from data analysis to workflow management to visualization tools. [ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030816">paper-2018</a>
</td>
          <td>
<a href="https://galaxyproject.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/nextflow-io/nextflow">Nextflow <img src="https://img.shields.io/github/stars/nextflow-io/nextflow" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/nextflow-io/nextflow" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> (recommended)</strong> - A fluent DSL modelled around the UNIX pipe concept, that simplifies writing parallel and scalable pipelines in a portable manner. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29412134">paper-2018</a>
</td>
          <td>
<a href="http://nextflow.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/insitro/redun">redun <img src="https://img.shields.io/github/stars/insitro/redun" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/insitro/redun" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A python-based workflow manager.</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/cgat-developers/ruffus">Ruffus <img src="https://img.shields.io/github/stars/cgat-developers/ruffus" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/cgat-developers/ruffus" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Computation Pipeline library for python widely used in science and bioinformatics. [ <a href="https://pubmed.ncbi.nlm.nih.gov/20847218">paper-2010</a>
</td>
          <td>
<a href="http://www.ruffus.org.uk">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/scipipe/scipipe">SciPipe <img src="https://img.shields.io/github/stars/scipipe/scipipe" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/scipipe/scipipe" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Workflow library embedded in the Go programming language, focusing on supporting complex workflow constructs, compiling to a single binary, providing powerful file naming and comprehensive audit reports for every output [ <a href="https://pubmed.ncbi.nlm.nih.gov/31029061/">paper-2019</a>
</td>
          <td>
<a href="https://scipipe.org/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/SeqWare/seqware">SeqWare <img src="https://img.shields.io/github/stars/SeqWare/seqware" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/SeqWare/seqware" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Hadoop Oozie-based workflow system focused on genomics data analysis in cloud environments. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21210981">paper-2010</a>
</td>
          <td>
<a href="https://seqware.github.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://bitbucket.org/snakemake">Snakemake</a></strong> - A workflow management system in Python that aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29788404">paper-2018</a>
</td>
          <td>
<a href="https://snakemake.readthedocs.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/broadinstitute/wdl">Workflow Descriptor Language <img src="https://img.shields.io/github/stars/broadinstitute/wdl" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/broadinstitute/wdl" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Workflow standard developed by the Broad. [ <a href="https://software.broadinstitute.org/wdl">web</a> ]</li>
</ul>

<h3 id="pipelines">Pipelines</h3>

<ul>
  <li>
<strong><a href="https://github.com/pditommaso/awesome-pipeline">Awesome-Pipeline <img src="https://img.shields.io/github/stars/pditommaso/awesome-pipeline" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pditommaso/awesome-pipeline" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A list of pipeline resources.</li>
  <li>
<strong><a href="https://github.com/bactopia/bactopia/">Bactopia <img src="https://img.shields.io/github/stars/bactopia/bactopia" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/bactopia/bactopia" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A flexible pipeline, built with Nextflow, for the complete analysis of bacterial genomes. [ <a href="https://bactopia.github.io/">web</a> ]</li>
  <li>
<strong><a href="https://github.com/fmalmeida/bacannot">Bacannot <img src="https://img.shields.io/github/stars/fmalmeida/bacannot" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/fmalmeida/bacannot" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A generic but comprehensive bacterial annotation pipeline, built with Nextflow, with nice graphical options for investigating results. [ <a href="https://bacannot.readthedocs.io/en/latest/?badge=latest">web</a> ]</li>
  <li>
<strong><a href="https://github.com/chapmanb/bcbio-nextgen">bcbio-nextgen <img src="https://img.shields.io/github/stars/chapmanb/bcbio-nextgen" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/chapmanb/bcbio-nextgen" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Batteries included genomic analysis pipeline for variant and RNA-Seq analysis, structural variant calling, annotation, and prediction. [ <a href="https://bcbio-nextgen.readthedocs.io">web</a> ]</li>
  <li>
<strong><a href="https://github.com/pentalpha/r-peridot">R-Peridot <img src="https://img.shields.io/github/stars/pentalpha/r-peridot" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pentalpha/r-peridot" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Customizable pipeline for differential expression analysis with an intuitive GUI. [ <a href="http://www.bioinformatics-brazil.org/r-peridot">web</a> ]</li>
  <li>
<strong><a href="https://github.com/fmalmeida/ngs-preprocess">ngs-preprocess <img src="https://img.shields.io/github/stars/fmalmeida/ngs-preprocess" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/fmalmeida/ngs-preprocess" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A pipeline for preprocessing short and long sequencing reads, built with Nextflow. [ <a href="https://ngs-preprocess.readthedocs.io/en/latest/?badge=latest">web</a> ]</li>
</ul>

<h3 id="sequence-processing">Sequence Processing</h3>

<p>Sequence Processing includes tasks such as demultiplexing raw read data, and trimming low quality bases.</p>

<ul>
  <li>
<strong><a href="https://github.com/OpenGene/AfterQC">AfterQC <img src="https://img.shields.io/github/stars/OpenGene/AfterQC" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/OpenGene/AfterQC" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/28361673">paper-2017</a> ]</li>
  <li>
<strong><a href="https://github.com/s-andrews/FastQC">FastQC <img src="https://img.shields.io/github/stars/s-andrews/FastQC" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/s-andrews/FastQC" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A quality control tool for high throughput sequence data. [ <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">web</a> ]</li>
  <li>
<strong><a href="https://github.com/mdshw5/fastqp">Fastqp <img src="https://img.shields.io/github/stars/mdshw5/fastqp" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/mdshw5/fastqp" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - FASTQ and SAM quality control using Python.</li>
  <li>
<strong><a href="https://github.com/agordon/fastx_toolkit">Fastx Tookit <img src="https://img.shields.io/github/stars/agordon/fastx_toolkit" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/agordon/fastx_toolkit" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - FASTQ/A short-reads pre-processing tools: Demultiplexing, trimming, clipping, quality filtering, and masking utilities. [ <a href="http://hannonlab.cshl.edu/fastx_toolkit">web</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/ewels/MultiQC">MultiQC <img src="https://img.shields.io/github/stars/ewels/MultiQC" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ewels/MultiQC" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Aggregate results from bioinformatics analyses across many samples into a single report. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27312411">paper-2016</a>
</td>
          <td>
<a href="http://multiqc.info">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/telatin/seqfu2">SeqFu <img src="https://img.shields.io/github/stars/telatin/seqfu2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/telatin/seqfu2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Sequence manipulation toolkit for FASTA/FASTQ files written in Nim. [ <a href="https://www.mdpi.com/2306-5354/8/5/59">paper-2021</a>
</td>
          <td>
<a href="https://telatin.github.io/seqfu2/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/shenwei356/seqkit">SeqKit <img src="https://img.shields.io/github/stars/shenwei356/seqkit" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/shenwei356/seqkit" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Golang. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27706213">paper-2016</a>
</td>
          <td>
<a href="https://bioinf.shenwei.me/seqkit">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/fhcrc/seqmagick">seqmagick <img src="https://img.shields.io/github/stars/fhcrc/seqmagick" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/fhcrc/seqmagick" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - file format conversion in Biopython in a convenient way. [ <a href="http://seqmagick.readthedocs.io">web</a> ]</li>
  <li>
<strong><a href="https://github.com/lh3/seqtk">Seqtk <img src="https://img.shields.io/github/stars/lh3/seqtk" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/lh3/seqtk" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Toolkit for processing sequences in FASTA/Q formats.</li>
  <li>
<strong><a href="https://github.com/incertae-sedis/smof">smof <img src="https://img.shields.io/github/stars/incertae-sedis/smof" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/incertae-sedis/smof" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - UNIX-style FASTA manipulation tools.</li>
</ul>

<h3 id="data-analysis">Data Analysis</h3>

<p>The following items allow for scalable genomic analysis by introducing specialized databases.</p>

<ul>
  <li>
<strong><a href="https://github.com/hail-is/hail">Hail <img src="https://img.shields.io/github/stars/hail-is/hail" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/hail-is/hail" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Scalable genomic analysis.</li>
  <li>
<strong><a href="https://github.com/dnanexus-rnd/GLnexus">GLNexus <img src="https://img.shields.io/github/stars/dnanexus-rnd/GLnexus" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/dnanexus-rnd/GLnexus" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Scalable gVCF merging and joint variant calling for population sequencing projects. [ <a href="https://www.biorxiv.org/content/10.1101/343970v1.abstract">paper-2018</a> ]</li>
</ul>

<h3 id="sequence-alignment">Sequence Alignment</h3>

<h4 id="pairwise">Pairwise</h4>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/BenLangmead/bowtie2">Bowtie 2 <img src="https://img.shields.io/github/stars/BenLangmead/bowtie2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/BenLangmead/bowtie2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - An ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. [ <a href="https://pubmed.ncbi.nlm.nih.gov/22388286">paper-2012</a>
</td>
          <td>
<a href="http://bowtie-bio.sourceforge.net/bowtie2">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/lh3/bwa">BWA <img src="https://img.shields.io/github/stars/lh3/bwa" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/lh3/bwa" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Burrow-Wheeler Aligner for pairwise alignment between DNA sequences.</li>
  <li>
<strong><a href="https://github.com/smarco/WFA">WFA <img src="https://img.shields.io/github/stars/smarco/WFA" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/smarco/WFA" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - the wavefront alignment algorithm (WFA) which expoit sequence similarity to speed up alignment [ <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa777/5904262">paper-2020</a> ]</li>
  <li>
<strong><a href="https://github.com/jeffdaily/parasail">Parasail <img src="https://img.shields.io/github/stars/jeffdaily/parasail" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/jeffdaily/parasail" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - SIMD C library for global, semi-global, and local pairwise sequence alignments [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0930-z">paper-2016</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/mummer4/mummer">MUMmer <img src="https://img.shields.io/github/stars/mummer4/mummer" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/mummer4/mummer" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> -  A system for rapidly aligning entire genomes, whether in complete or draft form. [ <a href="http://mummer.sourceforge.net/MUMmer.pdf">paper-1999</a>
</td>
          <td><a href="http://mummer.sourceforge.net/MUMmer2.pdf">paper-2002</a></td>
          <td><a href="http://mummer.sourceforge.net/MUMmer3.pdf">paper-2004</a></td>
          <td>
<a href="http://mummer.sourceforge.net">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/bbuchfink/diamond">DIAMOND <img src="https://img.shields.io/github/stars/bbuchfink/diamond" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/bbuchfink/diamond" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - An ultrafast protein aligner for <code class="language-plaintext highlighter-rouge">blastp</code> and <code class="language-plaintext highlighter-rouge">blastx</code> like searches. [ <a href="https://www.nature.com/articles/s41592-021-01101-x">paper-2021</a> ]</li>
</ul>

<h4 id="multiple-sequence-alignment">Multiple Sequence Alignment</h4>

<ul>
  <li>
<strong><a href="https://github.com/ljdursi/poapy">POA <img src="https://img.shields.io/github/stars/ljdursi/poapy" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ljdursi/poapy" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Partial-Order Alignment for fast alignment and consensus of multiple homologous sequences. [ <a href="https://academic.oup.com/bioinformatics/article/18/3/452/236691">paper-2002</a> ]</li>
</ul>

<h4 id="clustering">Clustering</h4>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/soedinglab/MMseqs2">MMseqs2 <img src="https://img.shields.io/github/stars/soedinglab/MMseqs2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/soedinglab/MMseqs2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Ultra-fast, sensitive search and clustering suite for protein and nucleotide sequence sets. [ <a href="https://www.nature.com/articles/nbt.3988">paper-2017</a>
</td>
          <td>
<a href="https://www.nature.com/articles/s41467-018-04964-5">paper-2018</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h3 id="quantification">Quantification</h3>

<ul>
  <li>
<strong><a href="https://github.com/cole-trapnell-lab/cufflinks">Cufflinks <img src="https://img.shields.io/github/stars/cole-trapnell-lab/cufflinks" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/cole-trapnell-lab/cufflinks" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. [ <a href="https://www.nature.com/articles/nbt.1621">paper-2010</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/deweylab/RSEM">RSEM <img src="https://img.shields.io/github/stars/deweylab/RSEM" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/deweylab/RSEM" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A software package for estimating gene and isoform expression levels from RNA-Seq data. [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323">paper-2011</a>
</td>
          <td>
<a href="http://deweylab.github.io/RSEM/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h3 id="variant-calling">Variant Calling</h3>

<ul>
  <li>
<strong><a href="https://github.com/google/deepvariant">DeepVariant <img src="https://img.shields.io/github/stars/google/deepvariant" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/google/deepvariant" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Deep learning-based variant caller [ <a href="https://rdcu.be/7Dhl">paper-2018</a> ]</li>
  <li>
<strong><a href="https://github.com/ekg/freebayes">freebayes <img src="https://img.shields.io/github/stars/ekg/freebayes" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ekg/freebayes" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Bayesian haplotype-based polymorphism discovery and genotyping. [ <a href="http://arxiv.org/abs/1207.3907">web</a> ]</li>
  <li>
<strong><a href="https://github.com/broadgsa/gatk">GATK <img src="https://img.shields.io/github/stars/broadgsa/gatk" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/broadgsa/gatk" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Variant Discovery in High-Throughput Sequencing Data. [ <a href="https://software.broadinstitute.org/gatk">web</a> ]</li>
  <li>
<strong><a href="https://github.com/luntergroup/octopus">Octopus <img src="https://img.shields.io/github/stars/luntergroup/octopus" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/luntergroup/octopus" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A polymorphic bayesian genotyping model with wide applicability. [ <a href="https://www.nature.com/articles/s41587-021-00861-3">paper-2021</a> ]</li>
  <li>
<strong><a href="https://github.com/samtools/bcftools">bcftools <img src="https://img.shields.io/github/stars/samtools/bcftools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/samtools/bcftools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - samtools/bcftools are a suite of tools for manipulating NGS data and can be used to call variants. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19505943">paper-2009</a> | <a href="http://htslib.org">web</a> ]
    <h4 id="structural-variant-callers">Structural variant callers</h4>
  </li>
  <li>
<strong><a href="https://github.com/dellytools/delly">Delly <img src="https://img.shields.io/github/stars/dellytools/delly" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/dellytools/delly" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Structural variant discovery by integrated paired-end and split-read analysis. [ <a href="https://pubmed.ncbi.nlm.nih.gov/22962449">paper-2012</a> ]</li>
  <li>
<strong><a href="https://github.com/arq5x/lumpy-sv">lumpy <img src="https://img.shields.io/github/stars/arq5x/lumpy-sv" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/arq5x/lumpy-sv" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - lumpy: a general probabilistic framework for structural variant discovery. [ <a href="https://link.springer.com/article/10.1186/gb-2014-15-6-r84">paper-2014</a> ]</li>
  <li>
<strong><a href="https://github.com/Illumina/manta">manta <img src="https://img.shields.io/github/stars/Illumina/manta" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/Illumina/manta" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Structural variant and indel caller for mapped sequencing data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26647377">paper-2015</a> ]</li>
  <li>
<strong><a href="https://github.com/PapenfussLab/gridss">gridss <img src="https://img.shields.io/github/stars/PapenfussLab/gridss" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/PapenfussLab/gridss" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - GRIDSS: the Genomic Rearrangement IDentification Software Suite. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29097403">paper-2017</a> ]</li>
  <li>
<strong><a href="https://github.com/brentp/smoove">smoove <img src="https://img.shields.io/github/stars/brentp/smoove" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/smoove" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - structural variant calling and genotyping with existing tools, but,smoothly.</li>
</ul>

<h3 id="bam-file-utilities">BAM File Utilities</h3>

<ul>
  <li>
<strong><a href="https://github.com/pezmaster31/bamtools">Bamtools <img src="https://img.shields.io/github/stars/pezmaster31/bamtools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pezmaster31/bamtools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Collection of tools for working with BAM files. [ <a href="https://academic.oup.com/bioinformatics/article/27/12/1691/255399">paper-2011</a> ]</li>
  <li>
<strong><a href="https://github.com/AndersenLab/bam-toolbox">bam toolbox <img src="https://img.shields.io/github/stars/AndersenLab/bam-toolbox" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/AndersenLab/bam-toolbox" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> MtDNA:Nuclear Coverage; BAM Toolbox can output the ratio of MtDNA:nuclear coverage, a proxy for mitochondrial content.</li>
  <li>
<strong><a href="https://github.com/DarwinAwardWinner/mergesam">mergesam <img src="https://img.shields.io/github/stars/DarwinAwardWinner/mergesam" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/DarwinAwardWinner/mergesam" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Automate common SAM &amp; BAM conversions.</li>
  <li>
<strong><a href="https://github.com/brentp/mosdepth">mosdepth <img src="https://img.shields.io/github/stars/brentp/mosdepth" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/mosdepth" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing. [ <a href="https://pubmed.ncbi.nlm.nih.gov/29096012/">paper-2017</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/TimoLassmann/samstat">SAMstat <img src="https://img.shields.io/github/stars/TimoLassmann/samstat" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/TimoLassmann/samstat" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Displaying sequence statistics for next-generation sequencing. [ <a href="https://academic.oup.com/bioinformatics/article/27/1/130/201972">paper-2010</a>
</td>
          <td>
<a href="http://samstat.sourceforge.net">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/brentp/somalier">Somalier <img src="https://img.shields.io/github/stars/brentp/somalier" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/somalier" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs. [ <a href="https://pubmed.ncbi.nlm.nih.gov/32664994">paper-2020</a> ]</li>
  <li>
<strong><a href="https://github.com/zd1/telseq">Telseq <img src="https://img.shields.io/github/stars/zd1/telseq" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/zd1/telseq" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Telseq is a tool for estimating telomere length from whole genome sequence data. [ <a href="https://academic.oup.com/nar/article/42/9/e75/1249448">paper-2014</a> ]</li>
</ul>

<h3 id="vcf-file-utilities">VCF File Utilities</h3>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/samtools/bcftools">bcftools <img src="https://img.shields.io/github/stars/samtools/bcftools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/samtools/bcftools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Set of tools for manipulating VCF files. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26826718">paper-2016</a>
</td>
          <td><a href="https://pubmed.ncbi.nlm.nih.gov/28205675">paper-2017</a></td>
          <td>
<a href="http://samtools.github.io/bcftools">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/brentp/vcfanno">vcfanno <img src="https://img.shields.io/github/stars/brentp/vcfanno" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/vcfanno" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Annotate a VCF with other VCFs/BEDs/tabixed files. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27250555">paper-2016</a> ]</li>
  <li>
<strong><a href="https://github.com/vcflib/vcflib">vcflib <img src="https://img.shields.io/github/stars/vcflib/vcflib" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/vcflib/vcflib" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A C++ library for parsing and manipulating VCF files.</li>
  <li>
<strong><a href="https://github.com/vcftools/vcftools">vcftools <img src="https://img.shields.io/github/stars/vcftools/vcftools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/vcftools/vcftools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - VCF manipulation and statistics (e.g. linkage disequilibrium, allele frequency, Fst). [ <a href="https://pubmed.ncbi.nlm.nih.gov/21653522">paper-2011</a> ]</li>
</ul>

<h3 id="gff-bed-file-utilities">GFF BED File Utilities</h3>

<ul>
  <li>
<strong><a href="https://github.com/NBISweden/AGAT">AGAT <img src="https://img.shields.io/github/stars/NBISweden/AGAT" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/NBISweden/AGAT" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Suite of tools to handle gene annotations in any GTF/GFF format. [ <a href="https://agat.readthedocs.io/en/latest/?badge=latest">web</a> ]</li>
  <li>
<strong><a href="https://github.com/daler/gffutils">gffutils <img src="https://img.shields.io/github/stars/daler/gffutils" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/daler/gffutils" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - GFF and GTF file manipulation and interconversion. [ <a href="http://daler.github.io/gffutils">web</a> ]</li>
  <li>
<strong><a href="https://bedops.readthedocs.io/en/latest/index.html">BEDOPS</a></strong> - The fast, highly scalable and easily-parallelizable genome analysis toolkit. [ <a href="https://academic.oup.com/bioinformatics/article/28/14/1919/218826">paper-2012</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/arq5x/bedtools2">Bedtools2 <img src="https://img.shields.io/github/stars/arq5x/bedtools2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/arq5x/bedtools2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A Swiss Army knife for genome arithmetic. [ <a href="https://pubmed.ncbi.nlm.nih.gov/20110278">paper-2010</a>
</td>
          <td><a href="https://pubmed.ncbi.nlm.nih.gov/25199790">paper-2014</a></td>
          <td>
<a href="https://bedtools.readthedocs.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h3 id="variant-simulation">Variant Simulation</h3>

<ul>
  <li>
<strong><a href="https://github.com/adamewing/bamsurgeon">Bam Surgeon <img src="https://img.shields.io/github/stars/adamewing/bamsurgeon" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/adamewing/bamsurgeon" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Tools for adding mutations to existing <code class="language-plaintext highlighter-rouge">.bam</code> files, used for testing mutation callers. [ <a href="https://popmodels.cancercontrol.cancer.gov/gsr/packages/bamsurgeon">web</a> ]</li>
  <li>
<strong><a href="https://github.com/lh3/wgsim">wgsim <img src="https://img.shields.io/github/stars/lh3/wgsim" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/lh3/wgsim" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - <strong>Comes with samtools!</strong> - Reads simulator. [ <a href="https://popmodels.cancercontrol.cancer.gov/gsr/packages/wgsim">web</a> ]</li>
</ul>

<h3 id="variant-predictionannotation">Variant Prediction/Annotation</h3>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/teamdfir/sift">SIFT <img src="https://img.shields.io/github/stars/teamdfir/sift" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/teamdfir/sift" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Predicts whether an amino acid substitution affects protein function. [ <a href="https://pubmed.ncbi.nlm.nih.gov/12824425">paper-2003</a>
</td>
          <td>
<a href="http://sift.jcvi.org">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/pcingola/SnpEff">SnpEff <img src="https://img.shields.io/github/stars/pcingola/SnpEff" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pcingola/SnpEff" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Genetic variant annotation and effect prediction toolbox. [ <a href="https://www.tandfonline.com/doi/full/10.4161/fly.19695">paper-2012</a>
</td>
          <td>
<a href="https://pcingola.github.io/SnpEff">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://anaconda.org/bioconda/ensembl-vep">Ensembl VEP</a></strong> - The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. [ <a href="https://doi.org/10.1186/s13059-016-0974-4">paper-2016</a>
</td>
          <td>
<a href="http://www.ensembl.org/info/docs/tools/vep/index.html">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h3 id="python-modules">Python Modules</h3>

<h4 id="data">Data</h4>

<ul>
  <li>
<strong><a href="https://github.com/brentp/cruzdb">cruzdb <img src="https://img.shields.io/github/stars/brentp/cruzdb" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/cruzdb" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Pythonic access to the UCSC Genome database. [ <a href="https://academic.oup.com/bioinformatics/article/29/23/3003/248468">paper-2013</a> ]</li>
  <li>
<strong><a href="https://github.com/openvax/pyensembl">pyensembl <img src="https://img.shields.io/github/stars/openvax/pyensembl" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/openvax/pyensembl" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Pythonic Access to the Ensembl database. [ <a href="https://pyensembl.readthedocs.io/en/latest/pyensembl.html">web</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/cokelaer/bioservices">bioservices <img src="https://img.shields.io/github/stars/cokelaer/bioservices" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/cokelaer/bioservices" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Access to Biological Web Services from Python. [ <a href="https://academic.oup.com/bioinformatics/article/29/24/3241/194040">paper-2013</a>
</td>
          <td>
<a href="http://bioservices.readthedocs.io">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>

<h4 id="tools">Tools</h4>

<ul>
  <li>
<strong><a href="https://github.com/arq5x/cyvcf">cyvcf <img src="https://img.shields.io/github/stars/arq5x/cyvcf" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/arq5x/cyvcf" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A port of <a href="https://github.com/jamescasbon/PyVCF">pyVCF <img src="https://img.shields.io/github/stars/jamescasbon/PyVCF" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/jamescasbon/PyVCF" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> using Cython for speed.</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/brentp/cyvcf2">cyvcf2 <img src="https://img.shields.io/github/stars/brentp/cyvcf2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/brentp/cyvcf2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Cython + HTSlib == fast VCF parsing; even faster parsing than pyVCF. [ <a href="https://pubmed.ncbi.nlm.nih.gov/28165109">paper-2017</a>
</td>
          <td>
<a href="https://brentp.github.io/cyvcf2">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/daler/pybedtools">pyBedTools <img src="https://img.shields.io/github/stars/daler/pybedtools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/daler/pybedtools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Python wrapper for <a href="https://github.com/arq5x/bedtools">bedtools <img src="https://img.shields.io/github/stars/arq5x/bedtools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/arq5x/bedtools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a>. [ <a href="https://pubmed.ncbi.nlm.nih.gov/21949271">paper-2011</a>
</td>
          <td>
<a href="http://daler.github.io/pybedtools">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/mdshw5/pyfaidx">pyfaidx <img src="https://img.shields.io/github/stars/mdshw5/pyfaidx" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/mdshw5/pyfaidx" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Pythonic access to FASTA files.</li>
  <li>
<strong><a href="https://github.com/pysam-developers/pysam">pysam <img src="https://img.shields.io/github/stars/pysam-developers/pysam" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/pysam-developers/pysam" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Python wrapper for <a href="https://github.com/samtools/samtools">samtools <img src="https://img.shields.io/github/stars/samtools/samtools" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/samtools/samtools" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a>. [ <a href="https://pysam.readthedocs.io/en/latest/api.html">web</a> ]</li>
  <li>
<strong><a href="https://github.com/jamescasbon/PyVCF">pyVCF <img src="https://img.shields.io/github/stars/jamescasbon/PyVCF" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/jamescasbon/PyVCF" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A VCF Parser for Python. [ <a href="http://pyvcf.readthedocs.org/en/latest/index.html">web</a> ]</li>
</ul>

<h3 id="assembly">Assembly</h3>
<ul>
  <li>
<strong><a href="https://github.com/ablab/spades">SPAdes <img src="https://img.shields.io/github/stars/ablab/spades" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ablab/spades" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - SPAdes (St. Petersburg genome assembler) is an assembly toolkit containing various assembly pipelines and the de-facto standard for prokaryotic genome assemblies.</li>
  <li>
<strong><a href="https://github.com/ncbi/SKESA">SKESA <img src="https://img.shields.io/github/stars/ncbi/SKESA" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ncbi/SKESA" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - SKESA is a de-novo sequence read assembler for microbial genomes. It uses conservative heuristics and is designed to create breaks at repeat regions in the genome. This leads to excellent sequence quality without significantly compromising contiguity.</li>
</ul>

<h3 id="annotation">Annotation</h3>
<ul>
  <li>
<strong><a href="https://github.com/tseemann/prokka">Prokka <img src="https://img.shields.io/github/stars/tseemann/prokka" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/tseemann/prokka" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Prokka: rapid prokaryotic genome annotation. Prokka is one of the most cited annotation command line tools for microbial genome annotations.</li>
  <li>
<strong><a href="https://github.com/oschwengers/bakta">Bakta <img src="https://img.shields.io/github/stars/oschwengers/bakta" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/oschwengers/bakta" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes &amp; plasmids. It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis.</li>
</ul>

<h2 id="long-read-sequencing">Long-read sequencing</h2>

<h3 id="long-read-assembly">Long-read Assembly</h3>

<ul>
  <li>
<strong><a href="https://github.com/marbl/canu">canu <img src="https://img.shields.io/github/stars/marbl/canu" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/marbl/canu" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A single molecule sequence assembler for genomes large and small.</li>
  <li>
<strong><a href="https://github.com/fenderglass/Flye">flye <img src="https://img.shields.io/github/stars/fenderglass/Flye" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/fenderglass/Flye" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - De novo assembler for single molecule sequencing reads using repeat graphs.</li>
  <li>
<strong><a href="https://github.com/chhylp123/hifiasm">hifiasm <img src="https://img.shields.io/github/stars/chhylp123/hifiasm" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/chhylp123/hifiasm" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A haplotype-resolved assembler for accurate Hifi reads.</li>
  <li>
<strong><a href="https://github.com/ruanjue/wtdbg2">wtdbg2 <img src="https://img.shields.io/github/stars/ruanjue/wtdbg2" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ruanjue/wtdbg2" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> -  A fuzzy Bruijn graph approach to long noisy reads assembly</li>
</ul>

<h2 id="visualization">Visualization</h2>

<h3 id="genome-browsers--gene-diagrams">Genome Browsers / Gene Diagrams</h3>

<p>The following tools can be used to visualize genomic data or for constructing customized visualizations of genomic data including sequence data from DNA-Seq, RNA-Seq, and ChIP-Seq, variants, and more.</p>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/Lab41/squiggle">Squiggle <img src="https://img.shields.io/github/stars/Lab41/squiggle" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/Lab41/squiggle" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Easy-to-use DNA sequence visualization tool that turns FASTA files into browser-based visualizations. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30247632">paper-2018</a>
</td>
          <td>
<a href="https://squiggle.readthedocs.io/en/latest/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/dasmoth/dalliance">biodalliance <img src="https://img.shields.io/github/stars/dasmoth/dalliance" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/dasmoth/dalliance" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Embeddable genome viewer. Integration data from a wide variety of sources, and can load data directly from popular genomics file formats including bigWig, BAM, and VCF.
[ <a href="https://pubmed.ncbi.nlm.nih.gov/21252075">paper-2011</a> | <a href="http://www.biodalliance.org">web</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/biojs/biojs">BioJS <img src="https://img.shields.io/github/stars/biojs/biojs" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/biojs/biojs" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - BioJS is a library of over hundred JavaScript components enabling you to visualize and process data using current web technologies. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25075290/">paper-2014</a>
</td>
          <td>
<a href="http://biojs.net/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/jonathancrabtree/Circleator">Circleator <img src="https://img.shields.io/github/stars/jonathancrabtree/Circleator" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/jonathancrabtree/Circleator" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Flexible circular visualization of genome-associated data with BioPerl and SVG. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25075113">paper-2014</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/drio/dnaism">DNAism <img src="https://img.shields.io/github/stars/drio/dnaism" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/drio/dnaism" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Horizon chart D3-based JavaScript library for DNA data. [ <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-0891-2">paper-2016</a>
</td>
          <td>
<a href="http://drio.github.io/dnaism/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/igvteam/igv">IGV js <img src="https://img.shields.io/github/stars/igvteam/igv" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/igvteam/igv" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Java-based browser. Fast, efficient, scalable visualization tool for genomics data and annotations. Handles a large variety of formats. [ <a href="https://pubmed.ncbi.nlm.nih.gov/31099383">paper-2019</a>
</td>
          <td>
<a href="https://software.broadinstitute.org/software/igv">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/lairdm/islandplot">Island Plot <img src="https://img.shields.io/github/stars/lairdm/islandplot" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/lairdm/islandplot" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - D3 JavaScript based genome viewer. Constructs SVGs. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25916842/">paper-2015</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/GMOD/jbrowse">JBrowse <img src="https://img.shields.io/github/stars/GMOD/jbrowse" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/GMOD/jbrowse" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - JavaScript genome browser that is highly customizable via plugins and track customizations. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27072794">paper-2016</a>
</td>
          <td>
<a href="http://jbrowse.org/">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/chgibb/PHAT">PHAT <img src="https://img.shields.io/github/stars/chgibb/PHAT" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/chgibb/PHAT" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Point and click, cross platform suite for analysing and visualizing next-generation sequencing datasets. [ <a href="https://pubmed.ncbi.nlm.nih.gov/30561651">paper-2018</a>
</td>
          <td>
<a href="https://chgibb.github.io/PHATDocs">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/hammerlab/pileup.js">pileup.js <img src="https://img.shields.io/github/stars/hammerlab/pileup.js" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/hammerlab/pileup.js" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - JavaScript library that can be used to generate interactive and highly customizable web-based genome browsers. [ <a href="https://pubmed.ncbi.nlm.nih.gov/27153605">paper-2016</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/chmille4/Scribl">scribl <img src="https://img.shields.io/github/stars/chmille4/Scribl" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/chmille4/Scribl" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - JavaScript library for drawing canvas-based gene diagrams. [ <a href="https://pubmed.ncbi.nlm.nih.gov/23172864">paper-2012</a>
</td>
          <td>
<a href="http://chmille4.github.io/Scribl">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong>Lucid Align</strong> - A modern sequence alignment viewer. [ <a href="https://lucidalign.com">web</a> ]</li>
</ul>

<h3 id="circos-related">Circos Related</h3>

<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://github.com/vigsterkr/circos">Circos <img src="https://img.shields.io/github/stars/vigsterkr/circos" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/vigsterkr/circos" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - Perl package for circular plots, which are well suited for genomic rearrangements. [ <a href="https://pubmed.ncbi.nlm.nih.gov/19541911">paper-2009</a>
</td>
          <td>
<a href="http://circos.ca">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong>ClicO FS</strong> - An interactive web-based service of Circos. [ <a href="https://pubmed.ncbi.nlm.nih.gov/26227146">paper-2015</a> ]</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong>OmicCircos</strong> - R package for circular plots for omics data. [ <a href="https://pubmed.ncbi.nlm.nih.gov/24526832">paper-2014</a>
</td>
          <td>
<a href="http://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong>J-Circos</strong> - A Java application for doing interactive work with circos plots. [ <a href="https://pubmed.ncbi.nlm.nih.gov/25540184">paper-2014</a>
</td>
          <td>
<a href="http://www.australianprostatecentre.org/research/software/jcircos">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<strong><a href="https://bitbucket.org/henryhzhang/rcircos/src/master/">rCircos</a></strong> - R package for circular plots. [ <a href="https://pubmed.ncbi.nlm.nih.gov/23937229">paper-2013</a>
</td>
          <td>
<a href="http://watson.nci.nih.gov/cran_mirror/web/packages/RCircos/index.html">web</a> ]</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<strong><a href="https://github.com/mkanai/fujiplot">fujiplot <img src="https://img.shields.io/github/stars/mkanai/fujiplot" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/mkanai/fujiplot" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a></strong> - A circos representation of multiple GWAS results. [ <a href="https://www.nature.com/articles/s41588-018-0047-6">paper-2018</a> ]</li>
</ul>

<h2 id="database-access">Database Access</h2>

<ul>
  <li>
<a href="http://www.ncbi.nlm.nih.gov/books/NBK179288/">Entrez Direct: E-utilities on the UNIX command line</a> - UNIX command line tools to access NCBI’s databases programmatically. Instructions to install and examples are found in the link.</li>
</ul>

<h2 id="resources">Resources</h2>

<h3 id="becoming-a-bioinformatician">Becoming a Bioinformatician</h3>

<ul>
  <li><a href="http://blog.fejes.ca/?p=2418">What is a bioinformatician</a></li>
  <li><a href="http://www.ploscompbiol.org/article/info:doi%2F10.1371%2Fjournal.pcbi.1003496">Bioinformatics Curriculum Guidelines: Toward a Definition of Core Competencies</a></li>
  <li><a href="http://caseybergman.wordpress.com/2012/07/31/top-n-reasons-to-do-a-ph-d-or-post-doc-in-bioinformaticscomputational-biology/">Top N Reasons To Do A Ph.D. or Post-Doc in Bioinformatics/Computational Biology</a></li>
  <li>
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-1-104">A 10-Step Guide to Party Conversation For Bioinformaticians</a> - Here is a step-by-step guide on how to convey concepts to people not involved in the field when asked the question: ‘So, what do you do?’</li>
  <li>
<a href="https://www.youtube.com/watch?v=uwsjwMO-TEA">A History Of Bioinformatics (In The Year 2039)</a> - A talk by C. Titus Brown on his take of looking back at bioinformatics from the year 2039. His notes for this talk can be found <a href="http://ivory.idyll.org/blog/2014-bosc-keynote.html">here</a>.</li>
  <li>
<a href="https://madhadron.com/science/farewell_to_bioinformatics.html">A farewell to bioinformatics</a> - A critical view of the state of bioinformatics.</li>
  <li>
<a href="http://www.acgt.me/blog/2014/3/25/101-questions-a-new-series-of-interviews-with-notable-bioinformaticians">A Series of Interviews with Notable Bioinformaticians</a> - Dr. Keith Bradnam “thought it might be instructive to ask a simple series of questions to a bunch of notable bioinformaticians to assess their feelings on the current state of bioinformatics research, and maybe get any tips they have about what has been useful to their bioinformatics careers.”</li>
  <li>
<a href="https://github.com/ossu/bioinformatics">Open Source Society University on Bioinformatics <img src="https://img.shields.io/github/stars/ossu/bioinformatics" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/ossu/bioinformatics" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Solid path for those of you who want to complete a Bioinformatics course on your own time, for free, with courses from the best universities in the World.</li>
  <li>
<a href="http://rosalind.info/">Rosalind</a> - Rosalind is a platform for learning bioinformatics through problem solving.</li>
  <li>
<a href="http://www.opiniomics.org/a-guide-for-the-lonely-bioinformatician/">A guide for the lonely bioinformatician</a> - This guide is aimed at bioinformaticians, and is meant to guide them towards better career development.</li>
  <li><a href="https://doi.org/10.1093/bib/bby063">A brief history of bioinformatics</a></li>
</ul>

<h3 id="bioinformatics-on-github">Bioinformatics on GitHub</h3>

<ul>
  <li>
<a href="https://github.com/HussainAther/awesome-alternative-splicing">Awesome-alternative-splicing <img src="https://img.shields.io/github/stars/HussainAther/awesome-alternative-splicing" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/HussainAther/awesome-alternative-splicing" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - List of resources on alternative splicing including software, databases, and other tools.</li>
  <li>
<a href="https://github.com/opendilab/awesome-AI-based-protein-design">Awesome AI-based Protein Design <img src="https://img.shields.io/github/stars/opendilab/awesome-AI-based-protein-design" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/opendilab/awesome-AI-based-protein-design" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - A collection of research papers for AI-based protein design.</li>
</ul>

<h3 id="sequencing">Sequencing</h3>

<ul>
  <li>
<a href="https://youtu.be/6Is3W7JkFp8">Next-Generation Sequencing Technologies - Elaine Mardis (2014)</a> [1:34:35] - Excellent (technical) overview of next-generation and third-generation sequencing technologies, along with some applications in cancer research.</li>
  <li>
<a href="https://liorpachter.wordpress.com/seq/">Annotated bibliography of *Seq assays</a> - List of ~100 papers on various sequencing technologies and assays ranging from transcription to transposable element discovery.</li>
  <li>
<a href="http://www.illumina.com/content/dam/illumina-marketing/documents/applications/ngs-library-prep/ForAllYouSeqMethods.pdf">For all you seq… (PDF)</a> (3456x5471) - Massive infographic by Illumina on illustrating how many sequencing techniques work. Techniques cover protein-protein interactions, RNA transcription, RNA-protein interactions, RNA low-level detection, RNA modifications, RNA structure, DNA rearrangements and markers, DNA low-level detection, epigenetics, and DNA-protein interactions. References included.</li>
</ul>

<h3 id="rna-seq">RNA-Seq</h3>

<ul>
  <li>
<a href="https://www.biostars.org/p/52152/">Review papers on RNA-seq (Biostars)</a> - Includes lots of seminal papers on RNA-seq and analysis methods.</li>
  <li>
<a href="https://github.com/griffithlab/rnaseq_tutorial">Informatics for RNA-seq: A web resource for analysis on the cloud <img src="https://img.shields.io/github/stars/griffithlab/rnaseq_tutorial" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/griffithlab/rnaseq_tutorial" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Educational resource on performing RNA-seq analysis in the cloud using Amazon AWS cloud services. Topics include preparing the data, preprocessing, differential expression, isoform discovery, data visualization, and interpretation.</li>
  <li>
<a href="http://rnaseq.uoregon.edu/">RNA-seqlopedia</a> - RNA-seqlopedia provides an awesome overview of RNA-seq and of the choices necessary to carry out a successful RNA-seq experiment.</li>
  <li>
<a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8">A survey of best practices for RNA-seq data analysis</a> - Gives awesome roadmap for RNA-seq computational analyses, including challenges/obstacles and things to look out for, but also how you might integrate RNA-seq data with other data types.</li>
  <li>
<a href="https://www.youtube.com/watch?v=5NiFibnbE8o">Stories from the Supplement</a> [46:39] - Dr. Lior Pachter shares his stories from the supplement for well-known RNA-seq analysis software CuffDiff and <a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a> and explains some of their methodologies.</li>
  <li>
<a href="https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools">List of RNA-seq Bioinformatics Tools</a> - Extensive list on Wikipedia of RNA-seq bioinformatics tools needed in analysis, ranging from all parts of an analysis pipeline from quality control, alignment, splice analysis, and visualizations.</li>
  <li>
<a href="https://github.com/crazyhottommy/RNA-seq-analysis">RNA-seq Analysis <img src="https://img.shields.io/github/stars/crazyhottommy/RNA-seq-analysis" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/crazyhottommy/RNA-seq-analysis" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - <a href="https://github.com/crazyhottommy">@crazyhottommy</a>’s notes on various steps and considerations when doing RNA-seq analysis.</li>
</ul>

<h3 id="chip-seq">ChIP-Seq</h3>

<ul>
  <li>
<a href="https://github.com/crazyhottommy/ChIP-seq-analysis">ChIP-seq analysis notes from Tommy Tang <img src="https://img.shields.io/github/stars/crazyhottommy/ChIP-seq-analysis" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/crazyhottommy/ChIP-seq-analysis" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Resources on ChIP-seq data which include papers, methods, links to software, and analysis.</li>
</ul>

<h3 id="youtube-channels-and-playlists">YouTube Channels and Playlists</h3>

<ul>
  <li>
<a href="https://www.genome.gov/12514288/current-topics-in-genome-analysis-2016-course-syllabus-handouts-and-videos/">Current Topics in Genome Analysis 2016</a> - Excellent series of fourteen lectures given at NIH about current topics in genomics ranging from sequence analysis, to sequencing technologies, and even more translational topics such as genomic medicine.</li>
  <li>
<a href="https://www.youtube.com/user/GenomeTV">GenomeTV</a> - “GenomeTV is NHGRI’s collection of official video resources from lectures, to news documentaries, to full video collections of meetings that tackle the research, issues and clinical applications of genomic research.”</li>
  <li>
<a href="https://www.youtube.com/user/LeadingStrand">Leading Strand</a> - Keynote lectures from Cold Spring Harbor Laboratory (CSHL) Meetings. More on <a href="http://theleadingstrand.cshl.edu/">The Leading Strand</a>.</li>
  <li>
<a href="https://www.youtube.com/playlist?list=PLqLDR0CTP9_pboZCk6gR9Zn4kW7h9XWJI">Genomics, Big Data and Medicine Seminar Series</a> - “Our seminars are dedicated to the critical intersection of GBM, delving into ‘bleeding edge’ technology and approaches that will deeply shape the future.”</li>
  <li>
<a href="https://www.youtube.com/user/RafalabChannel/videos">Rafael Irizarry’s Channel</a> - Dr. Rafael Irizarry’s lectures and academic talks on statistics for genomics.</li>
  <li>
<a href="https://www.youtube.com/user/nihvcast">NIH VideoCasting and Podcasting</a> - “NIH VideoCast broadcasts seminars, conferences and meetings live to a world-wide audience over the Internet as a real-time streaming video.” Not exclusively genomics and bioinformatics video but many great talks on domain specific use of bioinformatics and genomics.</li>
</ul>

<h3 id="blogs">Blogs</h3>

<ul>
  <li>
<a href="http://www.acgt.me/">ACGT</a> - Dr. Keith Bradnam writes about this “thoughts on biology, genomics, and the ongoing threat to humanity from the bogus use of bioinformatics acroynums.”</li>
  <li>
<a href="http://www.opiniomics.org/">Opiniomics</a> - Dr. Mick Watson write on bioinformatics, genomes, and biology.</li>
  <li>
<a href="https://liorpachter.wordpress.com/">Bits of DNA</a> - Dr. Lior Pachter writes review and commentary on computational biology.</li>
  <li>
<a href="http://www.michaeleisen.org/blog/">it is NOT junk</a> - Dr. Michael Eisen writes “a blog about genomes, DNA, evolution, open science, baseball and other important things”</li>
  <li>
<a href="https://bioinfoperl.blogspot.com">#!/perl/bioinfo</a> - The Computational and Structural Biology group at EEAD-CSIC writes, in Spanish and English, about ideas and code for plant genomics, computational and structural biology problems.</li>
</ul>

<h3 id="miscellaneous">Miscellaneous</h3>

<ul>
  <li>
<a href="https://github.com/jtleek/genomicspapers/">The Leek group guide to genomics papers <img src="https://img.shields.io/github/stars/jtleek/genomicspapers" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/jtleek/genomicspapers" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Expertly curated genomics papers to get up to speed on genomics, RNA-seq, statistics (used in genomics), software development, and more.</li>
  <li>
<a href="https://doi.org/10.1371/journal.pcbi.1003662">A New Online Computational Biology Curriculum</a> - “This article introduces a catalog of several hundred free video courses of potential interest to those wishing to expand their knowledge of bioinformatics and computational biology. The courses are organized into eleven subject areas modeled on university departments and are accompanied by commentary and career advice.”</li>
  <li>
<a href="http://www.foo.be/docs/tpj/issues/vol1_2/tpj0102-0001.html">How Perl Saved the Human Genome Project</a> - An anecdote by Lincoln D. Stein on the importance of the Perl programming language in the Human Genome Project.</li>
  <li>
<a href="https://liacs.leidenuniv.nl/~hoogeboomhj/mcb/nature_primer.html">Educational Papers from Nature Biotechnology and PLoS Computational Biology</a> - Page of links to primers and short educational articles on various methods used in computational biology and bioinformatics.</li>
  <li>
<a href="https://peerj.com/collections/45-bioinformatics-software/">The PeerJ Bioinformatics Software Tools Collection</a> - Collection of tools curated by Keith Crandall and Claus White, aimed at collating the most interesting, innovative, and relevant bioinformatics tools articles in PeerJ.</li>
</ul>

<h2 id="online-networking-groups">Online networking groups</h2>

<ul>
  <li>
<a href="https://discord.com/invite/3uxbPns">Bioinformatics (on Discord)</a> - a Discord server for general bioinformatics</li>
  <li>
<a href="https://www.reddit.com/r/bioinformatics/comments/7ndwm1/rbioinformatics_slack_channel_and_an_open_call/">r-bioinformatics</a> - the official Slack workspace of r/bioinformatics (<a href="https://www.reddit.com/message/compose/?to=apfejes&amp;subject=Request%20to%20join%20the%20r/bioinformatics%20Slack%20group&amp;message=I%20would%20like%20to%20request%20to%20join%20the%20r/bioinformatics%20Slack%20group">send a direct message to apfejes on reddit</a>)</li>
  <li>
<a href="https://bioinformaticsgrx.es/">BioinformaticsGRX</a> - A community of bioinformaticians based in Granada, Spain</li>
  <li>
<a href="https://comunidadbioinfo.github.io/">Comunidad de Desarolladores de Software en Bioinformática</a> - A community of bioinformaticians centered in Latin America</li>
  <li>
<a href="https://combine.org.au/">COMBINE</a> - An Austrialian group for bioinformatics students</li>
</ul>

<h2 id="license">License</h2>

<p><a href="https://creativecommons.org/publicdomain/zero/1.0/"><img src="http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg" alt="CC0" referrerpolicy="no-referrer" loading="lazy"></a></p>


      
    </div>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.0/anchor.min.js" integrity="sha256-lZaRhKri35AyJSypXXs4o6OPFTbTmUoltBbDCbdzegg=" crossorigin="anonymous"></script>
    <script>anchors.add();</script>
  </body>
</html>
